lib/server/controllers/configuration.controller.js

```diff
@@ -0,0 +1,33 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+
+// eslint-disable-next-line valid-jsdoc
+const { ServicesConfig } = require('../../config/index.js');
+
+// eslint-disable-next-line valid-jsdoc
+/**
+ * Request to provide configuration to the frontend
+ */
+const getConfigurationHandler = (_, response) => {
+    // eslint-disable-next-line require-jsdoc
+    const stringify = (value) => typeof value === 'string' ? `'${value}'` : value;
+
+    response.status(200).type('.js').send([
+        'window.CONFIGURATION={',
+        `FLP_INFOLOGGER_URL: ${stringify(ServicesConfig.infologger.flp.url)},`,
+        `EPN_INFOLOGGER_URL: ${stringify(ServicesConfig.infologger.epn.url)},`,
+        '};',
+    ].join(''));
```

Comment:
I think it can be simplified:
1. extract conf to a separate object
2. call `JSON.stringify` on it and pass a result to sth like:
``response.status(200).type('.js').send(`window.CONFIGURATION=${JSON.stringify(conf)}`);``
note: `JSON.stringify` works with js class `URL` as I think you want here